VIDEO_6_SCRIPT.md

```diff
@@ -0,0 +1,55 @@
+- How to join the AsyncAPI community
+
+Hello everybody! I hope that you’re doing well. Welcome again to the AsyncAPI video, where we explain the project and its progress. Today we’ll talk about the AsyncAPI community.
+
+## Collective strength
+
+A motor is made up of many parts. They have different shapes, sizes, weight, thickness, function... but they are all important. None of them separately can take you anywhere. In fact, individually they could not propel you to advance a single centimeter. But, with all them working together, as a clockwork, they add strenght enough for taking you many miles away and travel around the world.
+
+Joining forces, working together, we go further. That's clear. Collective strength is always higher than the sum of individual efforts.
+
+But, how to find a space for everyone? Every social group needs some common ground to meet and develop its goals. Working, as well as playing music or enjoying free time, is primarily about sharing things, thoughts, challenges, obstacles, joy, sorrow... And, after all that, find a place of understanding and, hopefully, entertainment. 
+
+Collectively, challenges are easier to overcome; everything seems smoother and funnier.
+
+There is not much more to say about why in AsyncAPI we prefer to go along together. To work as a community. 
+
+## Working as a community
+
+A community is united by goals projected for the future, hoping to build something valuable. The projection of a reachable better place to be. Even slowly, if it goes far, as the italians say. 
+
+Beyond generating a sense of belonging, communities facilitate the exchange of information, knowledge, and experiences, contributing to their members' personal and professional development. 
+
+The communities are here to tackle and reverse the problems of physical ones. They somehow are very valuable not when shortening geographical and cultural distances, but also to restore the lack of social integration of groups discriminated within real communities. They are open and inclusive by definition.
+
+## AsyncAPI Initiative and its community
+
+As mentioned in previous videos, AsyncAPI Initiative provides spec and tools managed and driven by its community. It’s a community open to everyone who participates in the specification. Everyone is welcome. AsyncAPI is devoted to forming a diverse community, comprising many people from different races, ethnicities, and cultures.
+
+That's why AsyncAPI regularly participate in different mentoring programs to onboard open source unexperienced people to the project, to show commitment to mentorship and making community more and more inclusive.
+
+It’s mainly made up of developers, but it’s open to receiving all kinds of professionals who can contribute or collaborate with the project. Everyone has something to say. All kinds of ideas, opinions, inputs, or contributions are welcome. At the same time, all constructive criticism is taken into account to continue growing and improving.
```

Comment:
**The majority of the AsyncAPI community members are developers, but we remain open to...**